app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/progressive_disclosure.scala.html

```diff
@@ -33,7 +33,7 @@
             </span>
         </summary>
         <div id="@{id}-body">
-            @body.get
+            @Html(HtmlFormat.escape(body.get).toString.replace("\n", "<br/>"))
```

Comment:
Can we add a test for this into ProgressiveDisclosureViewSpec please :)

Reply:
I've added a test for this